NuRadioReco/utilities/muon_flux.py

```diff
@@ -0,0 +1,159 @@
+import numpy as np
+from NuRadioReco.utilities import units, io_utilities
+import scipy.interpolate as interpolate
+from scipy.integrate import quad
+from scipy.integrate import dblquad
+import os
+
+def get_flux(energy, file_surface_mu_flux='data/muon_flux_E_SIBYLL23c_GSF.pickle'):
+    '''
+    provides muon flux over the full zenith band as calculated with MCEq at a given energy. The CR model used
+    is the GlobalSplineFit and the hadronic interaction model is Sibyll 2.3c.
+
+    Parameters
+    ----------
+    energy: float
+        energy at which flux is calculated (in eV)
+    file_surface_mu_flux: string
+        file with model for muon flux
+
+    Returns
+    -------
+    flux of muon in NuRadio units. (GeV**-1 * m**-2 * ns**-1 sr**-1)
+    '''
+    abspath = os.path.dirname(os.path.abspath(__file__))
```

Comment:
One more comment before you merge - it's good that you added this abspath, so the script works regardless of which directory it's called from. It does mean a user who has a different `file_surface_mu_flux` will have trouble using the `get_flux` function. This may not be an issue if you think it makes sense to always have the data files in this directory, or if you don't care about this functionality. 

But a solution would be to define `abspath` at the module level, and change the default value for `file_surface_mu_flux` instead (i.e. to `file_surface_mu_flux=os.path.join(abspath, 'data/muon_flux_E...').

Reply:
Thanks, that's a good one. I will remember it for the next modules.